### ISSUE TITLE
Document what axe-core rules this addon uses?

### ISSUE THREAD

**Issue by bjankord** [storybooks/storybook-addon-a11y#16]
I know axe-core has different sets of rules it uses to test accessibility issues which can be determined programmatically.

| Tag Name | Accessibility Standard/Purpose |
| ---------- | ------------------------------- |
| wcag2a | WCAG 2.0 Level A| 
| wcag2aa | WCAG 2.0 Level AA| 
| section508 | Section 508| 
| best-practice | Best practices endorsed by Deque| 
| experimental | Cutting-edge techniques| 

I'm curious which ones storybook-addon-a11y, I'm guessing it uses the entire set?

cc/  @ndelangen

**Comment by ndelangen** [comment]
We're using
https://github.com/dequelabs/axe-core/blob/HEAD/doc/API.md#api-name-axea11ycheck

and not passing in any options, so it must be the default of that package `axe-core`.

cc/ @jbovenschen

**Comment by jbovenschen** [comment]
This add-on currently doesn't allow any options to be specified, it is a feature which would certainly improve the features for every user, because everyone does have it's own specific usecase. should be great to pass the settings for axe-core in the withA11y decorator.

**Comment by ndelangen** [comment]
Yes, good feature we will definitely add at some point!

Would love some help with it actually @bjankord, would you be able to spend a few hours and implement it?

**Comment by ndelangen** [comment]
FYI I have merged this repo into the monorepo!

Please do not open any issues or PR's directly on this repo, but instead open them here:
https://github.com/storybooks/storybook
